Translation/Loader/XliffLoader.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
I didn't check more places, but that one broke extract command as `simplexml_load_file` expects path to file and resource is given. It worked before because of the __toString() method but this is no case any longer.
@franmomu 
@goetas

Reply:
I'll take a look